scripts/upgrade

```diff
@@ -199,8 +199,15 @@ fi
 
 # Define a function to execute commands with `occ`
 exec_occ() {
+    # Backward compatibility to upgrade from NC14 or older version
+    if [ $current_major_version -lt 15 ]
+    then
+        NEXTCLOUD_PHP_VERSION="7.0"
```

Comment:
It's going to fail on buster, do we have to deal with it? (and maybe install php 7.0 for the migration in this case?)

Reply:
Considering there's people on jessie...
I think that's going to be a problem sooner or later...

Maybe in such situation we could consider a manual way around. Anyway, if that happen, we will know.